plugins/parcelport/mpi/parcelport_mpi.cpp

```diff
@@ -284,7 +284,7 @@ namespace hpx { namespace traits
 #else
                 "env = ${HPX_HAVE_PARCELPORT_MPI_ENV:"
                         "MV2_COMM_WORLD_RANK,PMIX_RANK,PMI_RANK,OMPI_COMM_WORLD_SIZE,"
-                        "ALPS_APP_PE"
+                        "ALPS_APP_PE,PMIX_RANK"
```

Comment:
Did this one sneak in by mistake? Looks like `PMIX_RANK` is already in the list anyway...